wrote_about_us/ockovani-p7.json

```diff
@@ -0,0 +1,26 @@
+{
+  "best_articles": [
+    {
+      "date": "2021-06-24",
+      "title": "Praha 7 naočkovala během pátečního odpoledne 2850 lidí",
+      "url": "https://www.praha7.cz/praha-7-naockovala-behem-patecniho-odpoledne-2850-lidi/",
+      "img": "ockovani.webp",
+      "source": "Praha 7",
+      "tags": [
+        "article",
+        "covid19"
+      ]
+    },
+    {
+      "date": "19.07.2021",
+      "title": "Očkování v Praze 7 pomůže také Lékařům bez hranic, obcím postiženým tornádem a bouřemi a dalším potřebným",
+      "url": "https://www.praha7.cz/ockovani-v-praze-7-pomuze-take-lekarum-bez-hranic-obcim-postizenym-tornadem-a-bouremi-a-dalsim-potrebnym/",
+      "img": "ockovani-druhe-kolo.webp",
+      "source": "Praha 7",
+      "tags": [
+        "article",
+        "covid19"
+      ]
+    }
+  ]
+}
```

Comment:
Pridal bych ti tiskovku kde slibuji prachy pro MSF

Reply:
Dalsi URLs:
https://www.blesk.cz/clanek/zpravy-koronavirus/682182/praha-7-znovu-ockuje-po-vlastni-ose-tentokrat-se-zameruje-na-mlade.html

Reply:
https://monitoring.anopress.cz/orig/MyOrig?h=77396631 to nevim kde vyslo :)

Reply:
https://www.prazskypatriot.cz/praha-7-pripravila-ockovani-pro-zajemce-od-16-let/

Reply:
https://monitoring.anopress.cz/orig/MyOrig?h=11709409

Reply:
https://monitoring.anopress.cz/orig/MyOrig?h=73910631

Reply:
https://twitter.com/prazskynoviny/status/1403005281643155457
https://www.prazskynoviny.cz/2021/06/10/praha-7-radnice-pripravila-ockovani-pro-zajemce-od-16-let-ve-dvou-pevnych-terminech/
https://www.prazskynoviny.cz/2021/06/19/aktuality-sobota-19-6-2021/

Reply:
https://www.praha7.cz/praha-7-naockovala-behem-patecniho-odpoledne-2850-lidi/

Reply:
https://www.prazskypatriot.cz/praha-7-naockovala-behem-patecniho-odpoledne-2850-lidi/

Reply:
https://monitoring.anopress.cz/orig/MyRec?h=83520516
respektive radsi 
 https://youtu.be/tQ_NJgeU8CU

Reply:
https://www.prazskynoviny.cz/2021/06/10/praha-7-radnice-pripravila-ockovani-pro-zajemce-od-16-let-ve-dvou-pevnych-terminech/

Reply:
pridano